catboost/private/libs/algo_helpers/error_functions.h

```diff
@@ -216,6 +216,43 @@ class TMultiRMSEError final : public TMultiDerCalcer {
     }
 };
 
+class TMultiRMSEErrorWithMissingValues final : public TMultiDerCalcer {
+public:
+    explicit TMultiRMSEErrorWithMissingValues()
+        : TMultiDerCalcer(EHessianType::Diagonal) {
+    }
+
+    void CalcDers(
+        TConstArrayRef<double> approx,
+        TConstArrayRef<float> target,
+        float weight,
+        TVector<double>* der,
+        THessianInfo* der2
+    ) const override {
+        const int dim = target.size();
+        for (auto i : xrange(dim)) {
+            if (std::isnan((float)target[i])) {
```

Comment:
conversion to float is redundant, `target[i]` is already `float`

Reply:
Fixed